code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -1164,15 +1168,15 @@
 			switch(strength)
 				if("low power stun bolts")
 					strength = "high power stun bolts"
-					charge_cost = 100
-					set_fire_delay(FIRE_DELAY_TIER_6 * 3)
+					charge_cost = 50
+					set_fire_delay(FIRE_DELAY_TIER_1)
```

Comment:
This is a pretty big change delay-wise, double checking that it's intentional w/ this PR

Reply:
intended